src/languages/languagestypes.h

```diff
@@ -38,6 +38,7 @@ struct Language
 {
     QString code;
     QString name;
+    Qt::LayoutDirection rtl = Qt::LeftToRight; // right to left
```

Comment:
Setting and comment don't match

Reply:
I meant the full form of rtl - right to left. But I see your point. I think I'll delete the comment

Reply:
I think better name  just `direction`

Reply:
I agree